.env.docker.test

```
@@ -0,0 +1,81 @@
+SUSPICIOUS_LOCATION_THRESHOLD=1000
+UNUSUAL_DEVICE_THRESHOLD=30
+
+# Rate Limiting Configuration (Relaxed for tests)
+RATE_LIMIT_WINDOW_MS=60000
+RATE_LIMIT_MAX_REQUESTS=1000
+LOGIN_RATE_LIMIT_WINDOW_MS=60000
+LOGIN_RATE_LIMIT_MAX_ATTEMPTS=10
+
+# S3/MinIO Configuration (Docker service names)
+S3_ENDPOINT=http://minio:9000
+S3_BUCKET=chemchat-test
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The S3_BUCKET key should go before the S3_ENDPOINT key

(UnorderedKey)